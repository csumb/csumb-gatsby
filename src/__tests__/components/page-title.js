import React from 'react'
import renderer from 'react-test-renderer'
import PageTitle from 'components/header/page-title'

describe('Breadcrumbs', () => {
  it('renders a page title', () => {
    const tree = renderer
      .create(<PageTitle layout="page">The page title</PageTitle>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders sub titles for sites', () => {
    const tree = renderer
      .create(
        <PageTitle layout="page" sub="Sub title">
          The page title
        </PageTitle>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders nothing for sites', () => {
    const tree = renderer
      .create(<PageTitle layout="site">The page title</PageTitle>)
      .toJSON()
    expect(tree).toBeNull()
  })
})
