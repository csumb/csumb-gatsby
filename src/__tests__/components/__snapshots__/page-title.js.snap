// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders a page title 1`] = `
<h1
  className="css-qi0b1h"
>
  The page title
</h1>
`;

exports[`Breadcrumbs renders sub titles for sites 1`] = `
<h1
  className="css-qi0b1h"
>
  The page title
  <small
    className="css-1e4qcvf"
  >
    Sub title
  </small>
</h1>
`;
